aws/data_source_aws_efs_access_point.go

```diff
@@ -0,0 +1,142 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/arn"
+	"github.com/aws/aws-sdk-go/service/efs"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/terraform-providers/terraform-provider-aws/aws/internal/keyvaluetags"
+)
+
+func dataSourceAwsEfsAccessPoint() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceAwsEfsAccessPointRead,
+
+		Schema: map[string]*schema.Schema{
+			"access_point_id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"file_system_arn": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"file_system_id": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"arn": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"owner_id": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"posix_user": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"gid": {
+							Type:     schema.TypeInt,
+							Computed: true,
+						},
+						"uid": {
+							Type:     schema.TypeInt,
+							Computed: true,
+						},
+						"secondary_gids": {
+							Type:     schema.TypeSet,
+							Elem:     &schema.Schema{Type: schema.TypeInt},
+							Set:      schema.HashInt,
```

Comment:
btw, @bflad should i leave this or is it redundant?

Reply:
I'm actually not sure with `HashInt` -- I know `HashString` is the default when working with `TypeString` (which is more common). I'd just leave it for now and we can always remove it in the future if extraneous. 👍